package.json

```diff
@@ -18,12 +18,26 @@
   "bin": {
     "resin": "./bin/resin"
   },
+  "pkg": {
+    "scripts": [
+      "node_modules/resin-sync/build/capitano/*.js",
+      "node_modules/resin-sync/build/sync/*.js"
+    ],
+    "assets": [
+      "build/auth/pages/*.ejs",
+      "node_modules/resin-discoverable-services/services/**/*"
+    ]
+  },
   "scripts": {
-    "build": "gulp build && tsc && npm run doc",
+    "prebuild": "rimraf build/ build-bin/",
```

Comment:
Should this also delete the `build-zip` folder?

Reply:
Oooh yes, good catch, done.